webpack.config.js

```diff
@@ -1,22 +1,56 @@
+var MiniCssExtractPlugin = require('mini-css-extract-plugin');
+var CssMinimizerPlugin = require('css-minimizer-webpack-plugin');
 var TerserPlugin = require('terser-webpack-plugin');
 var webpack      = require('webpack');
 
 module.exports = {
   mode    : 'production',
-  entry   : __dirname + '/index.js',
+  entry   : {
+    genoverse: __dirname + '/index.js'
+  },
   target  : [ 'web', 'es5' ],
-  output  : { filename: 'genoverse.min.js', path: __dirname + '/js' },
+  output  : {
+    filename: '[name].min.js',
+    path: __dirname + '/build',
```

Comment:
I had to add these two lines:
```
library: 'Genoverse',
libraryTarget: 'commonjs2'
```
As for some reason package doesn't export genoverse without it